galaxy.yml

```diff
@@ -9,7 +9,7 @@ namespace: aalaesar
 name: nextcloud
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 2.0.0
+version: 1.9.3
```

Comment:
I'm OK for merging now but the version will trigger a release again .
given the CI (that I still don't fully understand :p), if we put `1.8.0` here, will it skip the release ?

Reply:
Yes @aalaesar, I reverted to 1.8.0, so it will not try to do anything.